src/main/java/pl/shop/models/OrderDetail.java

```diff
@@ -0,0 +1,37 @@
+package pl.shop.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "orderdetails")
+public class OrderDetail {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @ManyToOne(
+            fetch = FetchType.LAZY,
+            cascade = CascadeType.ALL
+    )
+    @JoinColumn(name = "[order]", referencedColumnName = "[id]")
+    private Order order;
+
+    @OneToOne
+    @JoinColumn(name = "product", referencedColumnName = "id")
+    private Product product;
+
+    private BigDecimal quantity;
```

Comment:
Wydaje mi się, że quantity może być zwykłym intem.

Reply:
> Wydaje mi się, że quantity może być zwykłym intem.

Może faktycznie, chodź jeśli już zmieniać to może na Double, w razie gdyby klient chciał zamówić jakąś część całości.

Reply:
> > Wydaje mi się, że quantity może być zwykłym intem.
> 
> Może faktycznie, chodź jeśli już zmieniać to może na Double, w razie gdyby klient chciał zamówić jakąś część całości.

To chyba musimy ustalić co my w końcu sprzedajemy? xD Jeżeli nie będzie opcji zamówienia "części" czegoś, no to lepiej zostać przy int'cie zwykłym.